sdk/PrebidMobile.xcodeproj/project.pbxproj

```diff
@@ -451,7 +451,7 @@
 					};
 					C2B16E1A1EE1BE69006304C0 = {
 						CreatedOnToolsVersion = 8.3.2;
-						DevelopmentTeam = HR94J6TSB3;
+						DevelopmentTeam = 282JR47M4L;
```

Comment:
Which team is that?

Reply:
Is this even relevant?

Reply:
<img width="663" alt="screen shot 2018-02-14 at 11 59 06" src="https://user-images.githubusercontent.com/30284097/36200996-c360fc6a-117e-11e8-981b-28ebc9605140.png">

Reply:
willhaben Cem Yesil personal team